<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>java.io.IOException: Not connected Error</title>
    <link>http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378</link>
    <description>I&#39;ve double checked my installation and everything seems to be set up correctly, but even if I start a patch with just one module it throws an error and won&#39;t go live. 

Generate code complete
MODULE_PATHS=
compiling C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.cpp
linking C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.dbg.elf
C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.elf  :
section           size   addr
.text              224      0
.rodata            548      0
.init_array          0      0
.fini_array          0      0
.comment           128      0
.ARM.attributes     57      0
Total              957


Done compiling patch
java.io.IOException: Not connected
java.io.IOException: Not connected
	at axoloti.connection.USBBulkConnection_v2.transmitPacket1(USBBulkConnection_v2.java:415)
	at axoloti.connection.USBBulkConnection_v2.sendAndGetReplyPtr(USBBulkConnection_v2.java:677)
	at axoloti.connection.USBBulkConnection_v2.mem_alloc(USBBulkConnection_v2.java:797)
	at axoloti.connection.USBBulkConnection_v2.transmitStartLive(USBBulkConnection_v2.java:694)
	at axoloti.live.patch.PatchViewLive.lambda$goLive$9(PatchViewLive.java:314)
	at axoloti.job.JobProcessor.lambda$exec$0(JobProcessor.java:21)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)</description>
    
    <lastBuildDate>Tue, 11 Feb 2020 06:42:46 +0000</lastBuildDate>
    <category>Patching</category>
    <atom:link href="http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>java.io.IOException: Not connected Error</title>
        <dc:creator><![CDATA[@mafgar]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/mafgar">@mafgar</a> wrote:</p>
          <blockquote>
              <p>Just to follow up, I downgraded to the firmware before 2.0 and it works great.</p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378/2">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378/2</link>
        <pubDate>Tue, 11 Feb 2020 06:42:46 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-6378-2</guid>
        <source url="http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378.rss">java.io.IOException: Not connected Error</source>
      </item>
      <item>
        <title>java.io.IOException: Not connected Error</title>
        <dc:creator><![CDATA[@mafgar]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/mafgar">@mafgar</a> wrote:</p>
          <blockquote>
              <p>I've double checked my installation and everything seems to be set up correctly, but even if I start a patch with just one module it throws an error and won't go live. </p>

<p>Generate code complete<br>MODULE_PATHS=<br>compiling C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.cpp<br>linking C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.dbg.elf<br>C:\Users\mafgar\DOCUME~1\AXOLOT~1.0/build/xpatch.elf  :<br>section           size   addr<br>.text              224      0<br>.rodata            548      0<br>.init_array          0      0<br>.fini_array          0      0<br>.comment           128      0<br>.ARM.attributes     57      0<br>Total              957</p>

<p>Done compiling patch<br>java.io.IOException: Not connected<br>java.io.IOException: Not connected<br>	at axoloti.connection.USBBulkConnection_v2.transmitPacket1(USBBulkConnection_v2.java:415)<br>	at axoloti.connection.USBBulkConnection_v2.sendAndGetReplyPtr(USBBulkConnection_v2.java:677)<br>	at axoloti.connection.USBBulkConnection_v2.mem_alloc(USBBulkConnection_v2.java:797)<br>	at axoloti.connection.USBBulkConnection_v2.transmitStartLive(USBBulkConnection_v2.java:694)<br>	at axoloti.live.patch.PatchViewLive.lambda$goLive$9(PatchViewLive.java:314)<br>	at axoloti.job.JobProcessor.lambda$exec$0(JobProcessor.java:21)<br>	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)<br>	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)<br>	at java.lang.Thread.run(Thread.java:748)</p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378/1">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378/1</link>
        <pubDate>Mon, 03 Feb 2020 19:54:35 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-6378-1</guid>
        <source url="http://community.axoloti.com/t/java-io-ioexception-not-connected-error/6378.rss">java.io.IOException: Not connected Error</source>
      </item>
  </channel>
</rss>
